frontend/src/toolbar/experiments/WebExperimentTransformField.tsx

```diff
@@ -28,197 +26,66 @@ export function WebExperimentTransformField({
-                                            >
-                                                {value}
-                                            </LemonButton>
-                                        )
-                                    })}
-                                    <LemonDivider className="my-1" />
-                                    <LemonButton
-                                        fullWidth
-                                        type="tertiary"
-                                        icon={<IconPencil />}
-                                        onClick={() => {
-                                            setEditSelectorShowing(true)
-                                        }}
-                                    >
-                                        Edit selector
-                                    </LemonButton>
-                                </>
-                            ),
-                            placement: 'bottom',
-                            matchWidth: true,
-                        },
+                    onClick={() => {
+                        selectVariant(variant)
+                        inspectForElementWithIndex(
+                            variant,
+                            selectedElementType as ElementSelectorType,
+                            inspectingElement === transformIndex ? null : transformIndex
+                        )
                     }}
                 >
-                    {transform.selector ? 'Change element' : 'Select element'}
+                    {inspectingElement === transformIndex && selectedVariant === variant
+                        ? 'Selecting…'
+                        : 'Select element'}
                 </LemonButton>
+                <LemonSelect
+                    placeholder="Select element type"
+                    value={selectedElementType}
+                    options={Object.entries(ElementSelectorButtonTypes).map(([key, value]) => ({
+                        label: value,
+                        value: key,
+                    }))}
+                    onChange={(value) => {
+                        selectElementType(value as ElementSelectorType)
+                    }}
+                />
             </div>
-            {editSelectorShowing && (
-                <div className="mb-2">
-                    <LemonInput
-                        value={transform.selector}
-                        onChange={(value) => {
-                            if (experimentForm.variants) {
-                                const variants = { ...experimentForm.variants }
-                                variants[variant].transforms[transformIndex].selector = value
-                                setExperimentFormValue('variants', variants)
-                            }
-                        }}
-                        placeholder="HTML element selector"
-                    />
-                </div>
-            )}
```

Comment:
@jurajmajerik Did you intentionally remove the ability to edit the HTML element selector?

Reply:
https://github.com/PostHog/posthog/pull/28571